### ISSUE TITLE
osrm: Getting distances of -1 for walking with points far from network

### ISSUE THREAD

**Issue by tahini** [chairemobilite/transition#571]
I was randomly clicking on the map for transit routing when I got this configuration, which resulted in a the interface showing for alternative an obviously very long walking trip of -1 distance. This should not happen.

![Capture d’écran du 2023-02-22 15-02-43](https://user-images.githubusercontent.com/4398897/220746274-92e6757c-42bf-43b2-b0c8-9714e966c145.png)

**Comment by greenscientist** [comment]
Is the walking tab showing the same?

**Comment by tahini** [comment]
Yes, the walking tab shows the same, so we are not correctly handling some osrm return values
